internal/proxy/proxy.go

```diff
@@ -14,11 +17,13 @@ import (
 type Proxy struct {
 	settings      []setting
 	envConfigPath string
+	aptConfigPath string
 }
 
 type options struct {
 	root          string
 	envConfigPath string
+	aptConfigPath string
```

Comment:
thinking of those 2 options, do we really need to configure those paths, or only `root`?